README.md

```diff
@@ -1,5 +1,7 @@
 # depviz - dependency visualizer for the web
 
+> dependency visualizer for the web https://jbenet.github.io/depviz
```

Comment:
The H1 above already has “dependency visualizer for the web“.  I'm fine with it in the header or in this blockquote, but would rather not have the same text in both locations.